roofit/roofitcore/inc/RooAbsOptTestStatistic.h

```diff
@@ -91,6 +93,9 @@ class RooAbsOptTestStatistic : public RooAbsTestStatistic {
   Bool_t      _optimized ; //!
   double      _integrateBinsPrecision{-1.}; // Precision for finer sampling of bins.
 
+  bool _batchMode = false; //!
+  bool _originallyCacheOptimized = false;
```

Comment:
```suggestion
  double      _integrateBinsPrecision{-1.}; // Precision for finer sampling of bins.
  Bool_t      _optimized ; //!

  bool _batchMode = false; //!
  bool _originallyCacheOptimized = false;
```
Pack the bools into one location to reduce padding bytes. Remember that doubles need to be aligned to 8 bytes, so packing anything between 1 to 7 bools in front of them is bad.

Further, does `_originallyCacheOptimized` take part in I/O?
- If yes, you need to increment the class version.
- If no, remove it from I/O using `//!`

Reply:
I moved the bools to the end.

I now increased the class version and kept both `_batchMode` and `_originallyCacheOptimized` in the IO because the old `_batchedEvaluations` flag in RooNLLVar was also not removed from the IO.

Reply:
Let's maybe overthink this. The fewer things take part in I/O, the better. Otherwise, evolving the class becomes very cumbersome.
Can the state of those flags be reconstructed when a fit starts? If yes, remove those from I/O.